code/src/Album/Album.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./Album.css";
+import { Logos } from "../Logos";
+
+const Album = props => (
+  <div className="albumCard">
+    <div
+      className="albumImg"
+      style={{ backgroundImage: `url(${props.item.images[1].url})` }}
+    >
+      <div className="theLogos">
+        <Logos />
+      </div>
+    </div>
+
+    <div className="albumDetails">
+      <a className="albumName" href={props.item.external_urls.spotify}>
+        {props.item.name}
+      </a>
+      <a
+        className="artistName"
+        href={props.item.artists[0].external_urls.spotify}
+      >
+        {props.item.artists[0].name}
+      </a>
```

Comment:
Here, to be able to show multiple artists (if there are multiple artists on the album) do a similar solution as you did in App.js to get all the albums. 
```
{props.artist.map(artist => (
    <a className="artist" href={artist.external_urls.spotify}>{artist.name}</a>
))}
```